src/api/cpp/cvc5.h

```diff
@@ -4223,42 +4267,47 @@ class CVC5_EXPORT Solver
 
   /**
    * Get an abduct.
+   *
    * SMT-LIB:
-   * \verbatim
-   * ( get-abduct <conj> )
-   * \endverbatim
+   *
+   *     ( get-abduct <conj> )
+   *
    * Requires enabling option 'produce-abducts'
    * @param conj the conjecture term
-   * @param output a term C such that A^C is satisfiable, and A^~B^C is
-   *        unsatisfiable, where A is the current set of assertions and B is
-   *        given in the input by conj
-   * @return true if it gets C successfully, false otherwise
+   * @param output a term @f$C@f$ such that @f$A \wedge C@f$ is satisfiable,
+   *               and @f$A \wedge \neg B \wedge C@f$ is unsatisfiable, where
+   *               @f$A@f$ is the current set of assertions and @f$B@f$ is
+   *               given in the input by ``conj``
+   * @return true if it gets abduct @f$C@f$ successfully, false otherwise
    */
   bool getAbduct(const Term& conj, Term& output) const;
 
   /**
    * Get an abduct.
+   *
    * SMT-LIB:
-   * \verbatim
-   * ( get-abduct <conj> <g> )
-   * \endverbatim
+   *
+   *     ( get-abduct <conj> <g> )
+   *
    * Requires enabling option 'produce-abducts'
    * @param conj the conjecture term
-   * @param grammar the grammar for the abduct C
-   * @param output a term C such that A^C is satisfiable, and A^~B^C is
-   *        unsatisfiable, where A is the current set of assertions and B is
-   *        given in the input by conj
-   * @return true if it gets C successfully, false otherwise
+   * @param grammar the grammar for the abduct @f$C@f$
+   * @param output a term C such that @f$A \wedge @f$C@f$ is satisfiable, and
```

Comment:
I suspect that we have an odd number of `@f$` here...
```suggestion
   * @param output a term C such that @f$A \wedge C@f$ is satisfiable, and
```